android/app/src/main/java/com/expensify/chat/MainActivity.java

```diff
@@ -57,4 +59,32 @@ protected void onCreate(Bundle savedInstanceState) {
     }
     BootSplash.init(R.drawable.bootsplash, MainActivity.this); // <- display the generated bootsplash.xml drawable over our MainActivity
   }
+
+  /**
+   * This method is called when a key down event has occurred.
+   * Forwards the event to the KeyCommandModule
+   */
+  @Override
+  public boolean onKeyDown(int keyCode, KeyEvent event) {
+    // disabling hardware ESCAPE support which is handled by Android
+    if (event.getKeyCode() == KeyEvent.KEYCODE_ESCAPE) { return false; }
```

Comment:
Nab - This is weird style here, I'd go with a new line for all of these

```suggestion
    if (event.getKeyCode() == KeyEvent.KEYCODE_ESCAPE) { 
        return false; 
    }
```

Reply:
@azimgd seems like we have this small code style change to address